__tests__/modern.test.ts

```diff
@@ -24,12 +24,8 @@ it('Creates a simple css variable based theme', () => {
     `,
     `
       .test {
-        color: var(--color);
-        background-image: linear-gradient(to right, var(--color), var(--color))
-      }
-
-      :root {
```

Comment:
Remove the 'root' for all tests bc we are inlining the default.

Reply:
as per my review it would be cool if this was a config option